lib/ansible/cli/galaxy.py

```diff
@@ -363,6 +363,8 @@ def add_verify_options(self, parser, parents=None):
                                    'path/url to a tar.gz collection artifact. This is mutually exclusive with --requirements-file.')
         verify_parser.add_argument('-i', '--ignore-errors', dest='ignore_errors', action='store_true', default=False,
                                    help='Ignore errors during verification and continue with the next specified collection.')
+        verify_parser.add_argument('--no-download', dest='no_download', action='store_true', default=False,
```

Comment:
Without the Galaxy signing feature or a way to check the initial checksum against the server, the local manifest can be tampered with. I've hoped we would be able to remove the download entirely. It's so early in the cycle that I wonder if Galaxy might yet have bandwidth to help us fix the default code path before this would be in a release.
Anyway, if that's possible at some point, this flag name would be confusing since both codepaths would be 'no download' but not doing the same thing.

Reply:
I think that makes sense. Right now the fact that we download the collection is an implementation detail.  So something indicating `--local-only` or `--no-server-verification`.  Just some ideas.

Reply:
Can we call this `--local-integrity-only` then?

Reply:
This is actually going into 2.11 though, and that's why we display the manifest hash- it gives us the top of an out-of-band validation that the whole chain is the same.

I guess I could also name the arg like the internal impl (something like `--local-verify-only` or `--offline`), which would let us be circumspect about the fact that "consulting the server" today means "downloading the artifact" where in the future it might be just asking for the hash.

Thoughts?

Reply:
wfm